src/posit/connect/vanities.py

```diff
@@ -231,6 +225,7 @@ def find_vanity(self) -> Vanity:
         -------
         Vanity
         """
-        response = self.params.session.get(self._endpoint)
+        endpoint = self.params.url + f"v1/content/{self._content_guid}/vanity"
```

Comment:
It kinda feels weird to me to see this change from a computed property to a repeated format string 🤔

Reply:
Makes sense. Realistically, this is a temporary change since this module will be rewritten to use the Active bases.